### DISCUSSION TITLE
In AttributeReader.php line 116: Undefined constant "App\Entity\id"

### DISCUSSION THREAD

**Discussion by yass19944** [symfony/symfony#49200] [Q&A]
I can't create an entity when I run the command symfony console make:entity or php bin/console make:migration it shows me this error message " In AttributeReader.php line 116:

Undefined constant "App\Entity\id" "

I looked in the project, there is no AttributeReader.php or App\Entity\id I updated the CLI composer but it does not work

**Comment by ahundiak** [comment]
make:entity and make:migration are two rather different things.  I'm guessing you did some hand editing of an entity and introduced an error.

Start by creating a new Symfony project (matching your current Symfony version) and using make:entity to create a simple entity.  Verify the error goes away.  If you actually do get the same error then come back, post your Symfony and PHP versions as well as the generated entity code.

If the error does go away then figure out what you did in your broken application.  Or at least come back and offer a few details.

**Reply by yass19944** [reply]
I changed the php version from 5.2 to 5.4 so I changed the entity annotations manually

**Reply by lyrixx** [reply]
You are using PHP 5.4? this is not a mistake? If so, Symfony does not support this PHP version for age. You should consider upgrading ASAP.

**Comment by stof** [comment]
Please try running the command in verbose mode to have a stack trace.

**Reply by yass19944** [reply]
![erreur](https://user-images.githubusercontent.com/124265883/216844643-1d5e7aef-13b2-4e5e-b895-c2557f15efe6.png)

**Reply by lyrixx** [reply]
You missed the `-vvv` option
